### DISCUSSION TITLE
I have created a port for the STM32F4XX M Board

### DISCUSSION THREAD

**Discussion by PCB-Rob** [micropython/micropython#13544] [STM32 / Pyboard] [answered]
I have been testing my new port of MicroPython to the STM32F4XX M Board
https://stm32-base.org/boards/STM32F407VGT6-STM32F4XX-M.html
What is the path to getting it tested and posted to the official site ?
Or should I just make a GitHub page with my port ?

**Comment by jimmo** [comment, marked as answer]
You can make a PR to add the board definition directory to `ports/stm32/boards`
